Project.toml

```diff
@@ -7,7 +7,7 @@ DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 
 [compat]
-DataStructures = "0.11, 0.12, 0.13, 0.14, 0.15"
+DataStructures = "0.11, 0.12, 0.13, 0.14, 0.15, 0.17"
```

Comment:
Any particular reason for skipping v0.16?

Reply:
No, I just forgot there is a 0.16 in between. Now I also ran test for `DataStructures`  v0.16.1 and it passed.  v0.16 is added to the list.